Orange/widgets/model/owunivariateregression.py

```diff
@@ -22,6 +22,9 @@ class OWUnivariateRegression(OWBaseLearner):
     name = "Univariate Polynomial Regression"
     description = "Univariate regression with polynomial expansion."
     icon = "icons/UnivariateRegression.svg"
+    replaces = [
+        "Orange.widgets.regression.owunivariateregression.OWUnivariateRegression",
+    ]
```

Comment:
All right, I was unaware of this. The replaces is already [present](https://github.com/biolab/orange3-educational/blob/master/orangecontrib/educational/widgets/owpolynomialregression.py#L35), so no update needed.